package.json

```diff
@@ -77,9 +72,18 @@
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
     "enzyme-to-json": "^3.3.5",
-    "typescript": "^3.7.4"
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "^3.3.1",
+    "typescript": "^3.7.4",
+    "typestyle": "^2.0.4"
```

Comment:
I don't think these fall under `devDepencies`, as they are run-time dependencies in production.

Reply:
They are added as peer to make sure they will be installed by dependants of this package. This guarantees both packages are using the same version.

Reply:
I see. Thanks for clarifying that!